README.md

```diff
@@ -1,2 +1,49 @@
-# storage
-Lite &amp; fast micro PHP storage library that is **easy to use**.
+# Utopia Storage
+
+[![Build Status](https://travis-ci.org/utopia-php/ab.svg?branch=master)](https://travis-ci.com/utopia-php/storage)
+![Total Downloads](https://img.shields.io/packagist/dt/utopia-php/storage.svg)
+[![Discord](https://img.shields.io/discord/564160730845151244?label=discord)](https://appwrite.io/discord)
+
+Utopia Storage library is simple and lite library for managing application storage. This library is aiming to be as simple and easy to learn and use. This library is maintained by the [Appwrite team](https://appwrite.io).
+
+This library is part of the [Utopia Framework](https://github.com/utopia-php/framework) project.
+
+
+## Getting Started
+
+Install using composer:
+```bash
+composer require utopia-php/storage
+```
+
+```php
+<?php
+
+require_once '../vendor/autoload.php';
+
+use Utopia\Storage\Storage;
+
```

Comment:
Missing init of new Local device. When we have support for S3 please add that as well, and also mention that we support multiple storage adapters in the description